front/lib/api/assistant/feedback.ts

```diff
@@ -202,39 +294,46 @@ export async function deleteMessageFeedback(
     throw new Error("Unexpected `auth` without `workspace`.");
   }
 
+  if (!canAccessConversation(auth, conversation)) {
+    throw new Error("User cannot access conversation");
+  }
+
   const message = await Message.findOne({
     where: {
       sId: messageId,
       conversationId: conversation.id,
     },
+    include: [
+      {
+        model: AgentMessage,
+        as: "agentMessage",
+        include: [
+          {
+            model: AgentMessageFeedbackResource.model,
```

Comment:
Sequelize JOINs are never good news. Make sure that it's fast and properly indexed. Please share EXPLAIN ANALYZE queries in production.